website/docs/docs/getting-started/custom-ui.md

```diff
@@ -37,29 +37,34 @@ You can also pass a **partial** theme object that overrides properties in the
 
 <!-- prettier-ignore -->
 <NestedSandpack nestedProps={`    theme={{
-      palette: {
-        accent: "rebeccapurple",
-      },
-      syntax: {
-        tag: "#006400",
-        string: "rgb(255, 165, 0)",
-        plain: "tomato",
-      },
-    }}`}
+        palette: {
+          accent: "rebeccapurple",
+        },
+        syntax: {
+          tag: "#006400",
+          string: "rgb(255, 165, 0)",
+          plain: "tomato",
+        },
+      }}`}
   />
 
+:::success Sandpack Theme Builder
+Use can design and customize your own theme, among other Sandpack presets. [Try it now! ↗](https://sandpack.codesandbox.io/theme)
```

Comment:
Two tiny nitpicks:
`You can design...`  (not use)
Are there any other presets you can customize at this time and via that link? Otherwise it doesnt' make much sense to me to say 'among other Sandpack presets'